next.config.mjs

```diff
@@ -32,6 +42,13 @@ const config = {
         pathname: "/**/*",
         port: "",
       },
+      // for minio images
+      {
+        hostname: "minio",
```

Comment:
Move the hostname to environment variable

Reply:
Add S3_ENDPOINT env variable